app/services/cocina_support.rb

```diff
@@ -146,4 +169,11 @@ def self.event_date_for(cocina_object:, type:) # rubocop:disable Metrics/AbcSize
       date_params[:day] = date.day if dash_count == 2
     end
   end
+
+  def self.orcid_for(contributor:)
+    orcid = contributor.identifier&.find { |id| id.type == 'ORCID' }&.value
+    return unless orcid.presence
+
+    "https://orcid.org/#{orcid}"
```

Comment:
"https://orcid.org" shouldn't be hard-coded all over the place.

Reply:
Added to settings.